cli.sh

```diff
@@ -1192,7 +1189,7 @@ update_shared_hooks_in() {
 
         if [ -d "$SHARED_ROOT/.git" ]; then
             echo "* Updating shared hooks from: $SHARED_REPO"
-            PULL_OUTPUT=$(cd "$SHARED_ROOT" && git pull 2>&1)
+            PULL_OUTPUT=$(git --work-tree="$SHARED_ROOT" --git-dir="$SHARED_ROOT/.git" pull 2>&1)
```

Comment:
Same:
```suggestion
            PULL_OUTPUT=$(cd "$SHARED_ROOT" && git -c core.hooksPath=/dev/null pull 2>&1)
```

Reply:
here GIT_DIR is defined.
When you run `git hooks shared update` suddenly GIT_DIR is defined (perhaps from before `git hooks` invocations...